README.md

```diff
@@ -12,9 +12,9 @@ We welcome contributions to help us improve and complete the .NET docs. This is
 
 To contribute, see the [Projects for .NET Community Contributors](https://github.com/dotnet/docs/projects/35) for ideas. The [Contributing Guide](CONTRIBUTING.md) has instructions on procedures we use. 
 
-If you're interested in helping migrate existing code that targets the .NET Framework from [Code Gallery](https://code.msdn.microsoft.com) to .NET Core applications stored in our [samples repository](https://github.com/dotnet/samples) and downloadable from the [Samples Browser](https://docs.microsoft.com/samples/browse), see the [Code Gallery migration](https://github.com/dotnet/docs/projects/88) project. 
+If you're interested in helping migrate existing code that targets the .NET Framework from the retired code gallery to .NET Core applications stored in our [samples repository](https://github.com/dotnet/samples) and downloadable from the [Samples Browser](https://docs.microsoft.com/samples/browse), see the [Code Gallery migration](https://github.com/dotnet/docs/projects/88) project. The code gallery samples were moved to the [Microsoft Archive](https://github.com/microsoftarchive?q=msdn-code-gallery) organization.
 
-We anticipate that [Xamarin](https://docs.microsoft.com/xamarin), [Mono](http://docs.go-mono.com/?link=root%3a%2fclasslib) and [Unity](https://docs.unity3d.com/Manual/index.html) will also use this documentation.
+We anticipate that [Xamarin](https://docs.microsoft.com/xamarin), [Mono](http://docs.go-mono.com/?link=root%3a%2fclasslib), and [Unity](https://docs.unity3d.com/Manual/index.html) will also use this documentation.
```

Comment:
I don't think this is true. I'd remove this. Xamarin already has its own repo on docs.

```suggestion
```